safenode/src/node/api.rs

```diff
@@ -285,9 +297,32 @@ impl Node {
                 // This will validate all the necessary components of the spend.
                 let res = match self
                     .transfers
-                    .try_add(signed_spend, parent_tx, fee_ciphers, parent_spends)
+                    .try_add(
+                        signed_spend.clone(),
+                        parent_tx.clone(),
+                        fee_ciphers.clone(),
+                        parent_spends.clone(),
+                    )
                     .await
                 {
+                    Ok(()) => {
+                        warn!("Broadcasting valid spend: {:?}", signed_spend.dbc_id());
+                        let event = Event::ValidSpendReceived {
+                            spend: signed_spend,
+                            parent_tx,
+                            fee_ciphers,
+                            parent_spends,
+                        };
+                        match self.send_to_closest(&Request::Event(event)).await {
```

Comment:
First, `send_to_closest` actually wait on response by default (and even wait on ALL responses)
Second, request/response model itself also indicated a response to be expected on sending out a request

Both, as I understand, not required for a `broadcasting` here?
i.e. here actually we can just spawn a thread to send the stuff out to avoid potential slow_down or even blocking ?

Reply:
If we don't have an api for that, I would prefer that we raise an issue and a PR for that, and then refactor this method here to use it.

Reply:
> Is there an api for that?

I don't know there is an existing one.
But, you can have a `spawned thread` , like here https://github.com/maidsafe/safe_network/blob/main/safenode/src/node/api.rs#L142-L146